modules/react-components/src/components/input/copy-input-field.tsx

```diff
@@ -20,6 +20,7 @@ import { TestableComponentInterface } from "@wso2is/core/models";
 import classNames from "classnames";
 import React, { FunctionComponent, MouseEvent, ReactElement, useEffect, useRef, useState } from "react";
 import { Button, Icon, Input, Popup } from "semantic-ui-react";
+import { CommonUtils } from "@wso2is/core/dist/src/utils";
```

Comment:
```suggestion
import { CommonUtils } from "@wso2is/core/utils";
```

Reply:
Thanks! This is now fixed in commit 80d844a